packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/ManageTopicsSection/ManageTopicsSection.tsx

```diff
@@ -0,0 +1,103 @@
+import { notifyError } from 'client/scripts/controllers/app/notifications';
+import type Topic from 'client/scripts/models/Topic';
+import app from 'client/scripts/state';
+import {
+  useFetchTopicsQuery,
+  useUpdateFeaturedTopicsOrderMutation,
+} from 'client/scripts/state/api/topics';
+import { CWText } from 'client/scripts/views/components/component_kit/cw_text';
+import { CWModal } from 'client/scripts/views/components/component_kit/new_designs/CWModal';
+import { CWButton } from 'client/scripts/views/components/component_kit/new_designs/cw_button';
+import { EditTopicModal } from 'client/scripts/views/modals/edit_topic_modal';
+import DraggableTopicsList from 'client/scripts/views/modals/order_topics_modal/draggable_topics_list';
+import React, { useState } from 'react';
+import './ManageTopicsSection.scss';
+
+export const ManageTopicsSection = () => {
+  const getFilteredTopics = (rawTopics: Topic[]): Topic[] => {
+    const topics = rawTopics
+      .filter((topic) => topic.featuredInSidebar)
+      .map((topic) => ({ ...topic } as Topic));
+
+    if (!topics.length) return [];
+
+    if (!topics[0].order) {
+      return topics
+        .sort((a, b) => a.name.localeCompare(b.name))
```

Comment:
lets not use `.sort` on the main topics array as it would mutate it (looks like it comes from react query state).

Reply:
Done